protocols/identify/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-identify"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Nodes identifcation protocol for libp2p"
-version = "0.44.1"
+version = "0.45.0"
```

Comment:
We shouldn't need a minor version bump I think. Now that the enums are `non_exhaustive`, this shouldn't be a breaking change :)

Reply:
:heavy_check_mark: